### DISCUSSION TITLE
Slow parsing when uc=True

### DISCUSSION THREAD

**Discussion by JimKarvo** [seleniumbase/SeleniumBase#2457] [Q&A] [answered]
It seems that it takes about 5 seconds to get a single page when I use uc=True

`from seleniumbase import Driver
import time

url = "https://example.com"
driver = Driver(uc=True)
epoch_time = int(time.time())
print("starting...")
driver.uc_open_with_tab(url)
print("seconds to load the site:", int(time.time()) - epoch_time)
`

The problem is that the website it loading under 0.2seconds, but at terminal i get

`
seconds to load the site: 5
`

I have tried to load an unprotected site, the same behaviour.
It is possible to decrease somehow this "waiting" time?

**Comment by mdmintz** [comment, marked as answer]
When you don't need to bypass bot-detection on a specific website while using UC Mode, use `driver.default_get(url)` to load that URL normally:

https://github.com/seleniumbase/SeleniumBase/blob/fd9727ae61e93097a9899c6befa2ecc3dc52ab52/help_docs/method_summary.md?plain=1#L928

Regular page loads in UC Mode take longer because chromedriver disconnects from Chrome for a few seconds to prevent anti-bot services on that website from detecting Selenium.

You can also customize the `reconnect_time` if you need a longer disconnect than the value set by default when opening a URL in UC mode. Use `driver.uc_open_with_reconnect(url, reconnect_time=None)` for that:

https://github.com/seleniumbase/SeleniumBase/blob/fd9727ae61e93097a9899c6befa2ecc3dc52ab52/help_docs/method_summary.md?plain=1#L1014

**Comment by HumbleExit** [comment]
Fucking love SeleniumBase @mdmintz thanks for all the hard work brother
